.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,36 @@
+name: Deploy
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
Once we get the OK to make changes, let's rename this to `main` or something [per NUIT policy](https://www.it.northwestern.edu/about/it-projects/dei/glossary.html).